gems/sorbet-runtime/lib/types/enum.rb

```diff
@@ -158,6 +158,11 @@ def to_json(*args)
     serialize.to_json(*args)
   end
 
+  sig {params(args: T.untyped).returns(T.untyped)}
+  def as_json(*args)
+    serialize
```

Comment:
The serialized value might not be a string/symbol (or other JSON compatible type), so I think we need to call `as_json` on it, to give it a chance to serialize itself appropriately:

```suggestion
    serialize.as_json(*args)
```

Reply:
Hmm, that's a fair point. However, now the tests won't pass as is because `as_json` doesn't exist on `String` unless ActiveSupport is included...